packages/spec-core/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@typespec/spec-core",
```

Comment:
make sure to make every package with `private:true`

Reply:
Done